papers/DX_zstring_view.fiets

```diff
@@ -2,28 +2,51 @@ std::zstring_view
+aforementioned types whenever a null-terminated C-style string is needed. While most C++ code mostly interfaces with C++
+code, it is not uncommon to need to use operating system calls, C interfaces, third-party library APIs, or even C++
+standard library APIs which require null-terminated strings. Because of a lack of a desirable option for passing
+non-owned null-terminated strings, `std::string_view` parameters are none the less sometimes used today in cases where
+null-terminated strings are needed. Using `std::string_view::data` in this way is, needless to say, very bug-prone.
+
+For this reason, many C++ developers use custom `zstring_view` or `cstring_view` types which are guaranteed to be
+null-terminated. We propose standardizing this utility.
+
+# Previous Papers
+
 Starting in the original paper from Sept 2012 - Feb 2014 introducing string_view, the idea of zstring_view is already present [N3921](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2014/n3921.html#null-termination):
 
 > Another option would be to define a separate zstring_view class to represent null-terminated strings and let it decay to string_view when necessary. That's plausible but not part of this proposal.
 
-An attempt was made in Feb 2019 to concretely propose the type (renamed to cstring_view) with [P1402](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1402r0.pdf), but it failed to gain consensus at [Kona](https://wiki.edg.com/bin/view/Wg21kona2019/P1402). In fact, the discussion there concluded with "CONSENSUS: We will not pursue P1402R0 or this problem space.". It should be noted that this discussion was in LEWGI, and was not taken by full LEWG.
+An attempt was made in Feb 2019 to concretely propose the type (renamed to cstring_view) with [P1402](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1402r0.pdf), but it failed to gain consensus at [Kona](https://wiki.edg.com/bin/view/Wg21kona2019/P1402). In fact, the discussion there concluded with "CONSENSUS: We will not pursue P1402R0 or this problem space." It should be noted that this discussion was in LEWGI, and was not taken by full LEWG. It is also worth noting that contracts were briefly contemplated in the minutes as a way of addressing the problem, however, contracts can't check the null char safely as it's not part of the view. Similarly other runtime checks for string view null-termination are off the table.
 
 However, in 2024 we get [P3081 Core safety profiles for C++26](https://wg21.link/p3081), which contains as an aside a null-terminated `zstring_view` in chapter 8, noting:
 
-> "This is one of the commonly-requested features from the [GSL] library that does not yet have a std:: equivalent. It was specifically requested by several reviewers of this work.". 
+> This is one of the commonly-requested features from the [GSL] library that does not yet have a std:: equivalent. It was specifically requested by several reviewers of this work.
```

Comment:
`.".` looks a little surprising, I figured since this is a blockquote we could just omit the outer `"`'s